### DISCUSSION TITLE
any way to track java udf jvm memory usage

### DISCUSSION THREAD

**Discussion by jiayeZhu** [apache/doris#42658] [A - General / Q&A]
hi,
we want to learn how much memory our java udf consumes in jvm, is there any way to track how the jvm memory usage per query?

**Comment by ixzc** [comment]
there is no single monitor for udf jvm. you can get be jvm from this
curl http://be_host:webserver_port/metrics
be jvm support pr
https://github.com/apache/doris/pull/35023
